.idea/mtcars.iml

```diff
@@ -3,6 +3,7 @@
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$">
       <excludeFolder url="file://$MODULE_DIR$/apps/dash-oil-and-gas/venv" />
+      <excludeFolder url="file://$MODULE_DIR$/venv" />
```

Comment:
hidden files should be added into .gitignore and **never**  be pushed onto github, these include .idea/*, env, *.pyc, DS_Store and build files. Pycharm has .gitignore template you could add and commit together with your app.

Reply:
This .idea could be removed as well before we proceed on deploying it to dash-deployment-server.  Other than that everything looks good.